src/main/kotlin/com/zepben/evolve/database/sqlite/common/BaseDatabaseWriter.kt

```diff
@@ -70,7 +71,12 @@ abstract class BaseDatabaseWriter(
         return status and postSave()
     }
 
-    abstract fun saveSchema(): Boolean
+    /**
+     * Populate tables with entries determined by the model for this database writer.
+     *
+     * @return true if the model was saved successfully.
+     */
+    internal abstract fun populateTables(): Boolean
```

Comment:
Do I count this as a breaking change? Nothing outside the SDK inherits from `BaseDatabaseWriter`, so I reckon not.

Reply:
In reality it is unlikely, but this is public API, so yes it is a breaking change as you have no control over what other people outside Zepben could have done.

Reply:
Breaking change noted in changelog.

Reply:
You also changed it from a public function to an internal one which is a breaking change. Which is probably a good thing as I would imagine we don't want people calling it outside of the `save` function.
It could probably actually become `protected` rather than `internal` even.